dronekit/__init__.py

```diff
@@ -1678,6 +1710,13 @@ def listener(self, attr_name, value):
         """
         return self._location
 
+    @property
+    def wind(self):
+        """
+        Current wind status (:pu:class: `Wind`)
+        """
+        return Wind(self._wind_direction, self._wind_speed, self._wind_speed_z)
```

Comment:
Shouldn't this, like battery, return `None` if the attributes are all `None`?

Reply:
Done